src/main/java/adris/altoclef/tasks/misc/speedrun/BeatMinecraftTask.java

```diff
@@ -380,14 +438,14 @@ private Task netherTick(AltoClef mod) {
         }
 
         // Blaze rods
-        int powderCount = mod.getInventoryTracker().getItemCountIncludingTable(Items.BLAZE_POWDER) + getBlazeRodsInPosession(mod) * 2;
-        if (powderCount < TARGET_BLAZE_RODS * 2) {
+        int powderCount = mod.getInventoryTracker().getItemCountIncludingTable(Items.BLAZE_POWDER) + (getBlazeRodsInPosession(mod) << 1);
+        if (powderCount < TARGET_BLAZE_RODS << 1) {
```

Comment:
I know code optimizations can happen, but this ain't a compiler lmfao

Reply:
I mean,,, code optimization is still code optimization lol